Flutter/lib/LobbyScreen.dart

```diff
@@ -1,15 +1,12 @@
-import 'package:flutter/material.dart';
-import 'Animation.dart';
-import 'SettleScreen.dart';
-import 'Settle.dart';
-import 'app_localizations.dart';
+import 'import_all.dart';
 
 class LobbyScreen extends StatefulWidget {
   final Settle settle;
   final String userName;
   final bool isHost;
+  final DarkThemeProvider themeChange;
 
-  const LobbyScreen(this.settle, this.userName, this.isHost);
+  const LobbyScreen(this.settle, this.userName, this.isHost, this.themeChange);
```

Comment:
Again, seems more appropriate for DarkThemeProvider to be statically accessed

Reply:
Resolved in commit `app-style 2c993e9`